test/tests.js

```diff
@@ -41,6 +41,12 @@ QUnit.test('percent character in cookie value mixed with encoded values', functi
 	assert.strictEqual(Cookies.get('bad'), 'foo%bar"baz%bax=', 'should read the percent character');
 });
 
+QUnit.test('percent character in cookie value mixed with encoded values lower case', function (assert) {
+	assert.expect(1);
+	document.cookie = 'mybad=foo%bar%22baz%bax%3d';
```

Comment:
In `3d`, see [here](https://github.com/js-cookie/js-cookie/commit/6b66ed4e29dbfa1a223804cbdb8169e3ea636b52#commitcomment-12300704) for some context:

Although lowercase and uppercase digits are equivalent per spec, is there any browser where those characters are treated differently after being encoded with `encodeURIComponent`? Or what is the use case for considering lowercase characters in percent-encoding for the default decoding mechanism, can't this be done sing [converters](https://github.com/js-cookie/js-cookie#converters)?

```javascript
Cookies = Cookies.withConverter(acceptLowercaseEncoding);
```

We use the [Strictness Principle](https://medium.com/@fagnerbrack/the-strictness-principle-9997e483cafb) for new features in the library, therefore we need at least a reasonable use case that justifies increasing the scope of the behavior of the default decoding mechanism.